blocks/search-results-list-block/features/search-results-list/_children/search-result.jsx

```diff
@@ -100,8 +98,7 @@ const SearchResult = ({
           )}
           { (showDate || showByline) && (
             <div className="results-list--author-date">
-              { showByline && <Byline story={element} stylesFor="list" /> }
-              { showByline && showSeparator && showDate && <p className="dot-separator">&#9679;</p> }
+              { showByline && <Byline story={element} stylesFor="list" separator={showDate} /> }
```

Comment:
We want to ensure this `<Byline` component is the shared styles component being used not the arc block version